assets/cubyz/biomes/glacier/base.zig.zon

```diff
@@ -0,0 +1,90 @@
+.{
+	.properties = .{
+		.cold,
+		.wet,
+		.barren,
+		.ocean,
```

Comment:
If it's an ocean type biome, then in my opinion, it should be renamed to frozen ocean. It's not really a glacier at this point.